app/swagger/requests/messages/folders.rb

```diff
@@ -41,6 +41,15 @@ class Folders
                 key :'$ref', :Messages
               end
             end
+
+            # TODO: resolve 500 error that should be 400 error
+            response 400 do
```

Comment:
@mphprogrammer can you explain this comment you put in? What seems to be the problem here?